mcu/stm32f105.S

```diff
@@ -183,7 +183,22 @@ Reset_Handler:
     strb    r2, [r0, #RCC_APB2RSTR]
     strb    r2, [r0, #RCC_APB2ENR]
     cbz     r4, .L_start_boot
-.L_jump_to_app:
+
+#if ( (defined(DFU_VERIFY_CHECKSUM) && (DFU_VERIFY_CHECKSUM != _DISABLE) ) )
+
+    ldr     r1, =#_APP_START
+    ldr     r2, =__romend
+    sub     r2, r2, r1
+    sub     r2, #8
```

Comment:
As per conversation, the adjustment should use a define from the checksum routine to determine the length, this would allow the checksum to be replaced with SHA or some other more advanced routine.

Reply:
Fixed https://github.com/dmitrystu/sboot_stm32/pull/10/commits/0b643c2f37857561fb61ae8bd86335b32ecb72a4